src/main/webapp/index.html

```diff
@@ -1,12 +1,71 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Page Title</title>
+    <meta charset="UTF-8">
+    <title>Page Title</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDbqn_ZFgoyKA_oDqFDC1vJyBq7Z3m0SgM"></script>
```

Comment:
I don't think it's safe to commit the api key!
https://stackoverflow.com/questions/33313334/is-it-safe-to-share-on-github-my-google-maps-api-key#:~:text=3%20Answers&text=If%20you%20configure%20the%20referrers,a%20publicly%20available%20web%20site.&text=No%2C%20it%20not%20safe.

Reply:
@g-lilian As it is a javascript call, I have to embed the api_key inside the code at the current stage https://stackoverflow.com/questions/1364858/what-steps-should-i-take-to-protect-my-google-maps-api-key. I have set up the restrictions as required on the backend to only allow domain from our website. 
![image](https://user-images.githubusercontent.com/28522090/86516155-d9dcaa80-be50-11ea-861c-82d10427f8d8.png)

I will store the key in our datastore when it is set up.

Reply:
Ah I see, that's good to know :D